### PR TITLE
office validation

### PR DESCRIPTION
**Pull request by leo-bell**
Added validation on Add and Update Office pages.

### CODE REVIEW

File: src/lib/validations.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  // Validate name
+  if (!name) {
+    errors.name = "Name is required";
+  } else if (name.length > 15) {
+    errors.name = "Name should be no more than 15 characters";
+  }
+
+
+  // Validate type
+  console.log("OfficeType ", OfficeType)
```
Comment (grodriguez1983) [inline src/lib/validations.ts]: remove this console

File: src/components/Office/UpdateForm.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            name="name"
+            placeholder={office.name}
+            className="w-full px-5 py-1 text-gray-700 bg-gray-200 rounded h-10"
+            onChange={handleInputChange}
+          />
+          {errors.name && <p className="text-red-500 text-xs">{errors.name}</p>}
+        </div>
+        <div className="mb-4">
+          <label className="block text-sm mb-1">Type</label>
+          <select
```
Comment (grodriguez1983) [inline src/components/Office/UpdateForm.tsx:61]: this select looks like something re useable

Comment (leo-bell) [reply src/components/Office/UpdateForm.tsx:61]: Input and Select will be in a separate component so we can reuse them through the project. I will push that separately.

File: src/lib/validations.ts (hunk 1 above)
Comment (leo-bell) [reply src/lib/validations.ts]: Fixed it!

**Pull request merged**